components/automate-chef-io/content/docs/configuration.md

```diff
@@ -60,31 +62,32 @@ upgrade_strategy = "at-once"
 
 Then run `chef-automate config patch </path/to/your-file.toml>` to deploy your change.
 
-To upgrade a Chef Automate installation when the `upgrade_strategy` is set to `none`, run:
+To upgrade a Chef Automate installation with the `upgrade_strategy` set to `none`, run:
```

Comment:
This is the only of these cleanups that I'm unsure about.  The intent here is to say something like:

> If your Chef Automate installation's `upgrade_strategy` is set to `none`, you can still upgrade to the latest version by running:

I think "with" sounds strange to me here because the underlying command works the same regardless of upgrade_strategy and thus it also works "without" the upgrade strategy set to none.

Reply:
@stevendanna I re-wrote the sentence to get rid of the passive verb tense. Anything with "is + verb" - for example: "is set" - falls under passive verb tense. Would this be corrected by the following correction? I realized "the" could probably be removed for better flow:

> To upgrade a Chef Automate installation with `upgrade_strategy` set to `none`, run:

Reply:
I had assumed that is why you made the change.  My issue is not with the flow of the sentence (although I like the removal of "the"), but rather that we are being too subtle here about _why_ the user should care about the following command. Further, to some it might _imply_ that the following command might not work with an `upgrade_strategy` of `at-once` even though it does.  That said, I think it is best that we merge this change and I can open a PR to address my concerns if they continue to bother me.

The good news is that as part of this, I re-read this section and found that we overlooked the much more serious problem.  The command below is incorrect.  It should be

```bash
chef-automate upgrade run
```

Reply:
I am following your 🚋 of thought now. I'll have to think a bit further on this. 

In the meantime, let me incorporate the correct command. I am so glad you caught that!